tests/clients/test_fasjson.py

```diff
@@ -172,3 +172,46 @@ async def test_get_users_by_matrix_id_multiple_users(monkeypatch):
         ),
     ):
         await client.get_users_by_matrix_id("@cookie:biscuit.test")
+
+
+@pytest.mark.parametrize(
+    "username,expected_url",
+    [
+        ("biscuit_eater", "users/biscuit_eater/groups"),
+    ],
+)
+async def test_get_user_groups(monkeypatch, username, expected_url):
+    result = ["group1", "group2"]
+    client = FasjsonClient("http://fasjson.example.com")
+    mock__get = mock.AsyncMock(
+        return_value=httpx.Response(
+            200,
+            json={"groups": result},
+        )
+    )
+    monkeypatch.setattr(client, "_get", mock__get)
+
+    response = await client.get_user_groups(username)
+
+    groups = result
+    assert isinstance(groups, list)
```

Comment:
There's a bug here, groups is set to your expected result and not the actual result from the function under test (`response`)

Reply:
D

> There's a bug here, groups is set to your expected result and not the actual result from the function under test (`response`)

Thank you for noticing, done!